packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -66,6 +66,9 @@ abstract class BindingBase {
   static bool _debugInitialized = false;
   static bool _debugServiceExtensionsRegistered = false;
 
+  /// The window to which this binding is bound.
```

Comment:
Can you elaborate on this documentation? Consider someone who is new to Flutter, what questions might they have about this? Consider someone who is a Flutter expert, and imagine they had a question that led them to read this documentation; what might they want to know?

See writing prompts here: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc